.circleci/config.yml

```
@@ -11,14 +11,14 @@ jobs:
       - run:
           xmlsec1 --version
       - run:
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+          curl -sSL https://install.python-poetry.org | python -
```

```suggestion
          curl -sSL https://install.python-poetry.org | python - --version 1.3.2
```

Or 1.4.0 if you're so inclined and it works for you 🙈